packages/editor/src/components/browser/Browser.tsx

```diff
@@ -55,7 +55,7 @@ const Browser = ({ open, onOpenChange, types, accept, location, cmsOptions, role
 
   return (
     <>
-      <Dialog open={open} onOpenChange={onOpenChange}>
+      <Dialog open={open} onOpenChange={onOpenChange} modal={false}>
```

Comment:
ahhh currently not possible via the ui-components stuff... there all is attached to the body.
Btw, the problem in this case it the inscription dialog which is not attached to the body 🙈

Reply:
Yes exactly, had to change this to false, otherwise you couldn't put the focus on the popover.
All right, yes maybe we could improve the UI component in the future so that you can define the container of the popover.